### DISCUSSION TITLE
system restore

### DISCUSSION THREAD

**Discussion by edmundlaugasson** [elementary/triage#399] [Ideas]
Would propose to add as built-in feature the system restore, which can be achieved via [Timeshift](https://github.com/linuxmint/timeshift). This is Linux Mint app, which is also based on Ubuntu LTS. E.g. in Linux Mint it already is as built-in feature. [Actually Timeshift is mentioned already via one proposal here](https://github.com/orgs/elementary/discussions/343#discussioncomment-7257376).

**Comment by lenemter** [comment]
Timeshift has some technical flaws. It launches its entire gui as root user, which interferes with elementary OS dark style, so we can't use it in its current state.
